src/renderer/components/Config/PrereqsView.tsx

```diff
@@ -48,7 +48,7 @@ const PrereqsView = ({ sx }: Props) => {
   }
 
   const processDescriptions = (status: AppModel) => {
-    if (status.id === 'wsl' || status.id === 'wslUbuntu') {
+    if (status.id === 'wsl' || status.id === 'wslUbuntu' || status.id === 'wslUbuntuStore') {
```

Comment:
It will show installs docs for wsl (docs.Install_wsl)  common for all, - it is refer to the docs above